### PR TITLE
Issue #SB-13299 fix : removing hardcoded plugin version

### PR DESCRIPTION
**Pull request by shivashanmugam**
https://project-sunbird.atlassian.net/browse/SB-13299

### CODE REVIEW

**Comment by JagadishPujari** [comment]
@sivashanmugam-funtoot 
Why this object is declared 2 times
`org.ekstep.questionunit = org.ekstep.questionunit || {};`

File: org.ekstep.questionunit-1.1/renderer/components/js/components.js [hunk 1]
```diff
@@ -47,7 +47,6 @@ org.ekstep.questionunit.baseComponent = {
     }
 }
 
-org.ekstep.questionunit = org.ekstep.questionunit || {};
```
Comment (vinukumar-vs) [inline org.ekstep.questionunit-1.1/renderer/components/js/components.js]: Why did your removed this? if org.ekstep.questionunit object is not defined, then how can we define org.ekstep.questionunit.questionComponent object?

File: org.ekstep.questionunit-1.1/renderer/components/js/components.js [hunk 2]
```diff
@@ -107,14 +106,14 @@ org.ekstep.questionunit.questionComponent = {
...
-        org.ekstep.questionunit.questionComponent.loadAssetUrl($('#org-ekstep-questionunit-questionComponent-qimage'), $('#org-ekstep-questionunit-questionComponent-qimage').data('image'), 'org.ekstep.questionunit', '1.0');
+        var quesitonUnitPluginVer = PluginManager.pluginMap["org.ekstep.questionunit"].m.ver;
+        org.ekstep.questionunit.questionComponent.loadImageFromUrl($('#org-ekstep-contentrenderer-questionunit-questionComponent-downArwImg'), 'renderer/assets/down_arrow.png', 'org.ekstep.questionunit', quesitonUnitPluginVer);
+        org.ekstep.questionunit.questionComponent.loadImageFromUrl($('#org-ekstep-contentrenderer-questionunit-questionComponent-AudioImg'), 'renderer/assets/audio-icon.png', 'org.ekstep.questionunit', quesitonUnitPluginVer);
+        org.ekstep.questionunit.questionComponent.loadAssetUrl($('#org-ekstep-questionunit-questionComponent-qimage'), $('#org-ekstep-questionunit-questionComponent-qimage').data('image'), 'org.ekstep.questionunit', quesitonUnitPluginVer);
     }
 }
 jQuery.extend(org.ekstep.questionunit.questionComponent, org.ekstep.questionunit.baseComponent);
 
-org.ekstep.questionunit = org.ekstep.questionunit || {};
```
Comment (vinukumar-vs) [inline org.ekstep.questionunit-1.1/renderer/components/js/components.js]: The same as defined above comment.

File: org.ekstep.questionunit-1.1/renderer/components/js/components.js (hunk 1 above)
Comment (shivashanmugam) [reply org.ekstep.questionunit-1.1/renderer/components/js/components.js]: The below line was duplicated in multiple places in the same file
`org.ekstep.questionunit = org.ekstep.questionunit || {};`
Line number - 1
Line number - 50
Line number - 117
.
I have removed Line number 50 and 117 and kept Line number 1

**Pull request merged**